### PR TITLE
feat: translate all to indonesian

### PR DESCRIPTION
**Pull request by edikurniawan-dev**
I have transalate all to Indonesian

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [x] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

**Comment by li-jia-nan** [comment]
You need to update the snapshot:

`npm run test:update components/locale`

**Comment by afc163** [comment]
feature branch~

**Comment by yoyo837** [comment]
Personally, I think it’s okay to merge into master. It's for https://github.com/ant-design/ant-design/issues/23369.

**Comment by afc163** [comment]
ping @edikurniawan-dev

**Comment by afc163** [comment]
https://github.com/ant-design/ant-design/pull/48287#issuecomment-2039163832

**Comment by edikurniawan-dev** [comment]
sorry, where to update it?

**Comment by afc163** [comment]
Run `npm run test:update components/locale` in your machine, check the diff and commit it.

**Comment by afc163** [comment]
ping @edikurniawan-dev

**Comment by edikurniawan-dev** [comment]
i got this error when run `npm run test:update components/locale`
image

**Comment by afc163** [comment]
@edikurniawan-dev

Try `npm test -u components/locale`

File: components/locale/id_ID.ts [hunk 1]
```diff
@@ -1,48 +1,147 @@
...
+import type { Locale } from ".";
+import Calendar from "../calendar/locale/id_ID";
+import DatePicker from "../date-picker/locale/id_ID";
+import TimePicker from "../time-picker/locale/id_ID";
+
+const typeTemplate = "${label} tidak valid ${type}";
 
 const localeValues: Locale = {
-  locale: 'id',
+  locale: "id",
```
Comment (afc163) [inline components/locale/id_ID.ts:12]: use `'` instead of `"`

**Comment by edikurniawan-dev** [comment]
I have already run test in my local machine, and all passed

image

**Comment by afc163** [comment]
Please commit and push the snapshot changes on your machine.

**Comment by li-jia-nan** [comment]
In the `components/locale/__tests__/__snapshots__/index.test.tsx.snap`:

image

**Review by afc163 — APPROVED** [review]

**Pull request merged**